bigquery_storage/google/cloud/bigquery_storage_v1beta1/reader.py

```diff
@@ -131,6 +141,14 @@ def __iter__(self):
                     yield message
 
                 return  # Made it through the whole stream.
+            except google.api_core.exceptions.InternalServerError as exc:
+                resumable_error = False
+                for resumable_message in _STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES:
+                    resumable_error = (
+                        resumable_error or resumable_message in exc.message
+                    )
```

Comment:
It's IMO more readable to just use `any()`, removing the need to drag the intermediary `resumable_error` value through the entire loop.

```py
resumable_error = any(
    resumable_message in exc.message
    for resumable_message in _STREAM_RESUMPTION_INTERNAL_ERROR_MESSAGES
)